src/requirements/requirements.ts

```diff
@@ -5,14 +5,15 @@
 import { Uri, env } from 'vscode';
 import * as path from 'path';
 import * as fse from 'fs-extra';
-import * as expandHomeDir from 'expand-home-dir';
 import { Commands } from './commands';
 import { checkJavaPreferences } from './settings';
 import { findJavaHomes, getJavaVersion, JavaRuntime } from './findJavaRuntimes';
 
+const expandHomeDir = require('expand-home-dir');
```

Comment:
why is this changed to `require`?

Reply:
I don't exactly remember but I was getting some error which was solved by this. Also this way is used in readme of exapand-home-dir. https://www.npmjs.com/package/expand-home-dir